client/src/App.jsx

```diff
@@ -58,14 +384,15 @@ class App extends React.Component {
   getHouseData() {
```

Comment:
update client so client grabs the id from browser and then makes a request to the backend with that id